capture-sdk/sdk/src/main/java/net/gini/android/capture/tracking/useranalytics/tracker/AmplitudeUserAnalyticsEventTracker.kt

```diff
@@ -62,23 +60,69 @@ internal class AmplitudeUserAnalyticsEventTracker(
         trackEvent(eventName, emptySet())
     }
 
+    private val events: MutableList<AmplitudeEventModel> = mutableListOf()
+
     override fun trackEvent(
         eventName: UserAnalyticsEvent,
         properties: Set<UserAnalyticsEventProperty>
     ) {
         val superPropertiesMap = superProperties.associate { it.getPair() }
         val propertiesMap = properties.associate { it.getPair() }
         val finalProperties = superPropertiesMap.plus(propertiesMap)
+        val c: Calendar = Calendar.getInstance(TimeZone.getTimeZone("Europe/Berlin"))
 
-        amplitude.track(
-            eventType = eventName.eventName,
-            eventProperties = finalProperties
+        events.add(
+            AmplitudeEventModel(
+                userId = userIdProvider.getUserId(),
+                deviceId = installationIdProvider.getInstallationId(),
+                eventType = eventName.eventName,
+                time = c.timeInMillis,
+                platform = contextProvider.osName,
+                osVersion = contextProvider.osVersion,
+                deviceManufacturer = contextProvider.manufacturer,
+                deviceBrand = contextProvider.brand,
+                deviceModel = contextProvider.model,
+                versionName = contextProvider.versionName ?: "unknown",
+                osName = contextProvider.osName,
+                carrier = contextProvider.carrier ?: "unknown",
+                language = contextProvider.language,
+                eventProperties = finalProperties,
+                userProperties = userProperties,
+                appVersion = "1.0"
+            )
         )
 
+        Log.e("User journey", "Event: ${eventName.eventName}\n" +
+                properties.joinToString("\n") { "  ${it.getPair().first}=${it.getPair().second}" })
+
         LOG.debug("\nEvent: ${eventName.eventName}\n" +
                 properties.joinToString("\n") { "  ${it.getPair().first}=${it.getPair().second}" })
     }
 
+    fun startRepeatingJob(): Job {
+        return CoroutineScope(Dispatchers.IO).launch {
+            while (true) {
```

Comment:
[Request]
I think it will be better to have here the check like `while(isActive) {... }` becuase you need to remember that Coroutine can be canceled.